unittest/unit/update-bcd.ts

```diff
@@ -996,7 +996,7 @@ describe('BCD updater', () => {
             actual,
             bcdFromSupport({
               chrome: {version_added: '85', flags: [{}]},
-              chrome_android: false
+              chrome_android: 'mirror'
```

Comment:
@foolip I've changed this expected value since you last reviewed, and I'd like to know if you agree with my reasoning.

The unit test describes the condition where the "upstream" browser support is "yes, but behind a flag", and the data says the feature is absent from the "downstream" browser.

To my mind, that data is consistent with the upstream browser, so the mirroring is still in effect, and the `"mirror"` value ought to be preserved.

What do you think?